core/identity-management/iin-agent/test/utilsTest.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable no-unused-expressions */
+
+const fs = require("fs");
+const keyutil = require("jsrsasign").KEYUTIL;
+const sinon = require("sinon");
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+
+chai.use(chaiAsPromised);
+const { expect } = chai;
+chai.should();
+
+const { Wallets } = require("fabric-network");
+const { ContractImpl } = require("fabric-network/lib/contract");
+const { NetworkImpl } = require("fabric-network/lib/network");
+const membershipPb = require('@hyperledger-labs/weaver-protos-js/common/membership_pb');
+const agent_pb = require('@hyperledger-labs/weaver-protos-js/identity/agent_pb');
+
+const utils = require('../out/common/utils.js')
+
+function decodeMembershipJSON(membershipJSON) {
+    
```

Comment:
Why is this blank?

Reply:
At some point I thought to create such a function but then created another (`getForeignAttestedMembership`), then forgot to remove this.
Now removed.